ui/DesignSystem/Component/Remote.svelte

```diff
@@ -44,6 +44,10 @@
     <WithContext {data} name={context}>
       <slot {data} />
     </WithContext>
+  {:else if Array.isArray(data) && data.length === 0}
```

Comment:
Having a special handling for empty arrays might trip people up. It also limits the usability if we actually want to distinguish between the data not being there and an empty array. Is there a reason why we can’t have this guard at the site where `Remote` is used?

Reply:
The pattern I observed was that whenever we had lists it would go as follows `{#if list.length > 0} content {:else} EmptyState {/if}`. Ideally the empty should be encoded in the `remote.Data`, because then the owner of the remote store can decide when to transition into that state. Didn't want to do that refactoring just yet.

Reply:
Yes, it might make sense to extract the “if empty” pattern. But I don’t think that `Remote` is the right place for that. Since the pattern is quite small and not very complicated we could just leave it duplicated for now and tackle it properly later. I’ll leave it up to you, though.